CMakeLists.txt

```diff
@@ -397,6 +397,7 @@ set(
     program_options
     system
     unit_test_framework
+    date_time
```

Comment:
I believe the schedulers and a few other readers use some things from the date time, such as the gregorian calendar.

Reply:
Right. I was trying to trim the list, took too many off. Know if we're actually using timer or chrono? I see a TODO about timer in src/coupling/GhostSwingBusManager.cpp but no usage, and all the chrono I see is using std::chrono.

Reply:
I think I copied the list over from HELICS,  which uses timer, and chrono comes along with timer on some platforms

Reply:
Double checked, we definitely don't use boost.timer or boost.chrono. Can always add them back later if needed.